content/posts/April-2022.md

```diff
@@ -1,20 +1,40 @@
-  ([packit#1427](https://github.com/packit/packit-service/pull/1427)).
+  ([packit#1427](https://github.com/packit/packit-service/pull/1427))
 - Downstream synchronization of the Packit configuration file (aka `packit.yaml`) is now working properly again.
-  ([packit#1532](https://github.com/packit/packit/pull/1532)).
+  ([packit#1532](https://github.com/packit/packit/pull/1532))
 - `packit source-git update-dist-git` and `packit source-git update-source-git` now check the synchronization of source-git and dist-git repositories prior to doing the update. If the update can't be done, for example, because the histories have diverged, the command provides instructions on how to synchronize the repositories.
   A `--force` option is available to try to update the destination repository anyway.
-  ([packit#1534](https://github.com/packit/packit/pull/1534)).
+  ([packit#1534](https://github.com/packit/packit/pull/1534))
 - When using `post_upstream_clone` to generate your spec-file,
   Packit now correctly checks out a release before the action is run.
-  ([packit#1542](https://github.com/packit/packit/pull/1542)).
+  ([packit#1542](https://github.com/packit/packit/pull/1542))
+
+## Week 14 (April 5th - April 11th)
+
+- We have introduced two new build and test target aliases: `fedora-latest-stable` resolves to the latest stable Fedora
+  Linux release, while `fedora-branched` resolves to all branched releases (all Fedora Linux release, except `rawhide`).
+  ([packit#1546](https://github.com/packit/packit/pull/1546))
+- We have reverted the functionality of Packit that allowed you to have set **only** specific targets for Copr repositories.
+  This functionality was introduced a while ago and was found to be not very beneficial in cases of differently
+  configured Copr jobs building in parallel in the same Copr repository.
+  ([packit#1551](https://github.com/packit/packit/pull/1551))
+- If you are using our stage instance, we make it listen only on `/packit-stg` comment commands so you can now differentiate
+  between the instances when commanding Packit via pull-request or issue comments.
+  For the production instance, you can continue using `/packit` prefix as you are used to.
+  ([packit-service#1432](https://github.com/packit/packit-service/pull/1432))
+- A new configuration option `downstream_branch_name` has been added, which is meant to be used in source-git projects
+  and allows users to customize the name of the branch in dist-git which corresponds to the current source-git branch.
+  ([packit#1555](https://github.com/packit/packit/pull/1555))
+- `revision-range` argument of the source-git `update-source-git` command is now optional. If not specified,
+  dist-git commits with no counterpart in source-git are synchronized.
+  ([packit#1547](https://github.com/packit/packit/pull/1547))
```

Comment:
Do we want to cover source-git related changes as well? Or, make a dedicated part for `source-git`?

Reply:
I was thinking about this too, but we already cover the source-git related changes in our blogposts. Do we want to discuss this in the architecture?